common/src/main/java/org/terracotta/angela/common/TerracottaCommandLineEnvironment.java

```diff
@@ -66,10 +66,12 @@ private void validate(String javaVersion, Set<String> javaVendors, Set<String> j
     requireNonNull(javaVendors);
     requireNonNull(javaOpts);
 
-    javaVendors.stream()
-        .filter(((Predicate<String>) vendor -> vendor == null || vendor.isEmpty()).negate())
-        .findFirst()
-        .orElseThrow(() -> new IllegalArgumentException("None of the java vendors can be null or empty"));
+    if(!javaVendors.isEmpty()) {
```

Comment:
This change should not be needed if the `angela.java.vendor` property is correctly set to empty string (not null, empty).

Reply:
this is what I did.  I debugged it. There is an empty string, so the validation is failing.

Reply:
Seems like you didn't try ;-)
`TerracottaCommandLineEnvironment` at the top:

`Set<String> vendors = JAVA_VENDOR.getValue().equals("") ? new HashSet<>() : singleton(JAVA_VENDOR.getValue());`

So if javaVendors, no value is found and you end up in the orElseThrow()

Reply:
Note: the bug is also there for the java version